recipes/atpbar/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "atpbar" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d6fc797cddb96088d001cc4660450b083587a1a1f6eb99f9f02923d2d0987da3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - atpbar
+
+about:
+  home: https://github.com/alphatwirl/atpbar
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Progress bars for threading and multiprocessing tasks on terminal and Jupyter Notebook'
+  description: |
+    atpbar can display multiple progress bars simultaneously growing
+    to show the progresses of iterations of loops in threading or
+    multiprocessing tasks on terminal and Jupyter Notebook.
```

Personally I like including the dev and doc URLs even if they're the same:
```suggestion
    multiprocessing tasks on terminal and Jupyter Notebook.
  dev_url: https://github.com/alphatwirl/atpbar
  doc_url: https://github.com/alphatwirl/atpbar#atpbar
```

---

Ok. Thank you.